### DISCUSSION TITLE
Modify items with bad confidence

### DISCUSSION THREAD

**Discussion by doopie3** [Klocman/Bulk-Crap-Uninstaller#528] [Q&A] [answered]
The option to modify items with bad confidence is locked and i cannot enable it. i can enable it again if i reinstall bcuninstaller but it goes back again after a while. looking for a fix because constantly reinstalling is tiring

**Comment by Klocman** [comment, marked as answer]
The option is greyed out if there are no bad confidence items to show for a given application(s).
